CHANGES.rst

```diff
@@ -1,17 +1,24 @@
 Changes
 =======
 
-Version 0.9.1 (UNRELEASED)
+Version 0.9.1 (2023-09-27)
 --------------------------
 
-- Adds support for previewing PDF files in the workspace details page.
-- Adds an extra button for login with custom third-party Keycloak SSO authentication services.
+- Adds support for previewing PDF files present in a workflow's workspace.
+- Adds support for previewing ROOT files present in a workflow's workspace.
+- Adds support for logging in with a custom third-party Keycloak instance.
 - Adds a new menu item to the workflow actions popup to allow stopping running workflows.
+- Changes the workflow deletion message to clarify that attached interactive sessions are also closed when a workflow is deleted.
 - Changes the workflow progress bar to always display it as full for finished workflows.
-- Changes the interactive session notification by adding a message stating that the session will be closed after a specified number of days inactivity.
+- Changes the interactive session notification to also report that the session will be closed after a specified number of days of inactivity.
 - Changes the workflow-details page to make it possible to scroll through the list of workflow steps in the job logs section.
+- Changes the page headers to conform to sentence case.
+- Changes workspace file preview to support customisable file size limit.
```

Comment:
- Changes workspace file preview to support customisable maximum file size limit allowed for previewing.